client/src/components/FloatingButton/style.less

```diff
@@ -0,0 +1,90 @@
+@import '/src/themes/main.less';
```

Comment:
I believe this file is unused.

Reply:
Yeah, actually this whole file is unused.  It's from my work of previous project and I have convert less to css file.

Reply:
Fixed in https://github.com/Wiredcraft/test-fullstack/pull/78/commits/896ff392925122f9008b8c92aa14db82622a17a5